src/bun.js/bindings/bindings.cpp

```diff
@@ -2546,6 +2546,104 @@ JSC__JSValue JSC__JSValue__keys(JSC__JSGlobalObject* globalObject, JSC__JSValue
     RELEASE_AND_RETURN(scope, JSValue::encode(ownPropertyKeys(globalObject, object, PropertyNameMode::Strings, DontEnumPropertiesMode::Exclude)));
 }
 
+JSC__JSValue JSC__JSValue__values(JSC__JSGlobalObject* globalObject, JSC__JSValue objectValue)
```

Comment:
I like that this is the exact implementation of `Object.values`, but we shouldn't have duplicated code from JSC that can get out of sync. For now this is good enough, but in the future we will need to modify `ObjectConstructor.cpp/h` to expose the implementation of `Object.values`, similar to what is done for `Object.freeze/seal`